transform/transformers/CORATransformer.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/env python
+#   coding: UTF-8
+
+from collections import OrderedDict
+import enum
+import os.path
+import re
+import sys
+
+from transform.transformers.CSTransformer import CSTransformer
```

Comment:
CORATransformer inherits from CSTransformer in order to create zipfiles. Overrides other behaviour.